gluoncv/model_zoo/resnet.py

```diff
@@ -31,10 +31,13 @@
            'se_resnet101_v2', 'se_resnet152_v2',
            'get_resnet']
 
+import mxnet as mx
 from mxnet.context import cpu
 from mxnet.gluon.block import HybridBlock
 from mxnet.gluon import nn
 from mxnet.gluon.nn import BatchNorm
+from mxnet import use_np
+mx.npx.set_np()
```

Comment:
the numpy mode switch is perhaps not ready for master. I have created a branch: https://github.com/dmlc/gluon-cv/tree/numpy for numpy transition.

Reply:
Thank you for your review. I have changed the base to the NumPy branch.